### ISSUE TITLE
Plugin fails to load with SM error

### ISSUE THREAD

**Issue by poetatellus** [nosoop/SM-TFUtils#4]
Fresh install, attempted using both the newest and second newest release, plugin in plugins folder,  gamedata in gamedata file etc.
SM errors on attempting to load plugin:
```
[SM] Exception reported: Could not determine location to read CTFPlayer::m_aObjects from.
[SM] Blaming: tf2utils.smx
[SM] Call stack trace:
[SM] [0] SetFailState
[SM] [1] Line 243, scripting/tf2utils.sp::OnPluginStart
```
SM version 1.10.0.6528

**Comment by nosoop** [comment]
Hi &mdash; have you restarted your server? SourceMod 1.10 has an issue with caching missing gamedata, so if you didn't install `gamedata/tf2.utils.nosoop.txt` prior to the first load, it will produce that error.  (This has been fixed in 1.11, but restarting after ensuring all files are installed will work all the same.)

**Comment by poetatellus** [comment]
Yes, error persists over restarts

**Comment by nosoop** [comment]
Are you running a Windows server? The next possible issue is that a detour is overwriting the bytes of the signature being checked for.  If you also have the [Custom Attribute Starter Pack](https://github.com/nosoop/SM-TFCustomAttributeStarterPack) installed, `attr_sapper_reprograms_buildings` is a plugin that would cause that.

The possible solutions there are one of the following:

- Remove `attr_sapper_reprograms_buildings.smx` or other applicable plugin to prevent the patch from running in the first place, if you don't plan on using that plugin.
- Use SourceMod 1.11.0.6821 or newer &mdash; that includes the the PR that fixes this (alliedmodders/sourcemod#1642).  This isn't backported to 1.10 unfortunately.
- Update the gamedata to use a different, non-detoured signature.

I'll use an alternative signature for Windows that is less likely to be detoured to close the issue, but the other two would also be acceptable.

**Comment by nosoop** [comment]
The newest release (0.18.0.1) should resolve the issue; thanks for the report.
